lib/omniauth/strategies/bigcommerce.rb

```diff
@@ -30,8 +30,8 @@ class BigCommerce < OmniAuth::Strategies::OAuth2
       option :token_options, %i[scope context account_uuid]
       option :client_options,
              site: ENV.fetch('BC_AUTH_SERVICE', 'https://login.bigcommerce.com'),
-             authorize_url: '/oauth2/authorize',
-             token_url: '/oauth2/token'
+             authorize_url: 'oauth2/authorize',
+             token_url: 'oauth2/token'
```

Comment:
wouldn't this be a breaking change?

Reply:
Should not be. This is the same change done in the upstream gem. I do think we need a PoL before merge though.